projects/aws/eks-a-admin-image/provisioners/install_golang.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Why does the admin image need golang?

Reply:
Also, I figured it would not hurt as having go exposes ability to easily install go binaries via go install command.

Reply:
I thought this was supposed to serve the same usecase as the original snow image: eks-a admin machine to manage clusters in a disconnected scenario

It seems like what you are describing here is more like a eks-a dev machine.

Reply:
So main purpose for me is to serve as the EKS-A admin machine in CI. I don't have requirement of being disconnected as our lab has internet access. 

In addition to CI, I figured dev's can use same image as CI when interfacing with bare metal nodes in the lab for dev testing.